<!doctype html>
<html class="no-js" lang="">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="x-ua-compatible" content="ie=edge">
        <title>Biosystems People UCSF</title>
        <link rel="icon" type="images/png" href="img/favicon.ico">
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <link rel="apple-touch-icon" href="apple-touch-icon.png">
        <!-- Place favicon.ico in the root directory -->
      <link href="https://fonts.googleapis.com/css?family=Josefin+Slab|Neucha|Permanent+Marker" rel="stylesheet">

        <link rel="stylesheet" href="css/normalize.css">
        <link rel="stylesheet" href="css/main.css">
        <!-- Latest compiled and minified CSS -->
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">

        <!-- Optional theme -->
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css" integrity="sha384-fLW2N01lMqjakBkx3l/M9EahuwpSfeNvV63J5ezn3uZzapT0u7EYsXMjQV+0En5r" crossorigin="anonymous">
        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
        <link rel="stylesheet" type="text/css" href="slick/slick.css"/>
        <link rel="stylesheet" type="text/css" href="slick/slick-theme.css"/>
        <link rel="stylesheet" href="css/index.css">
        <link rel="stylesheet" href="css/people.css">
      </head>
      <body>
            <!--[if lt IE 8]>
                <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
            <![endif]-->
            <nav class="navbar navbar-inverse">
              <div class="container-fluid">
          <!-- Brand and toggle get grouped for better mobile display -->
          <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="index.html">Biosystems Group at UCSF</a>
          </div>

          <!-- Collect the nav links, forms, and other content for toggling -->
          <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav">
              <li><a href="index.html">Home</a></li>
              <li><a href="research.html">Research</a></li>
              <li class="active"><a href="people.html">People<span class="sr-only">(current)</span></a></li>
              <li><a href="#">Collaboration</a></li>
              <li><a href="#">Links</a></li>
              <li><a href="#">Observations</a></li>
              <li><a href="#">Publications</a></li>
              </li>
            </ul>
          </div><!-- /.navbar-collapse -->
        </div><!-- /.container-fluid -->
      </nav>
      <div class="container-fluid">
        <div class="row keywordRow">
          <div class="col-xs-12 wordFade">
              <div class="huntLabInterestsOne centered">
              <h2>modeling</h2>
              <h3>drug development</h3>
              <h2>computational biology</h2>
              <h4>mammalian tissue morphogenesis</h4>
              <h4>individualization of therapy</h4>
              <h3>bioinformatics</h3>
              <h3>comparing results</h3>
              <h3>gene database analysis</h3>
              <p>multi-agent</p>
              <h3>drug development</h3>
              <h2>tools</h2>
              <h4>malignancy</h4>
              <h4>simulation for cancer</h4>
              <h3>tumor spheroids</h3>
              <h3>application of engineering methods </h3>
              <h2>simulation</h2>
              <h3>protein auction determination</h3>

            </div>
            <div class="secondLab">
                  <h2>bioengineering</h2>
                  <h3>cell biology</h3>
                  <h2>engineering</h2>
                  <h2>mathematics</h2>
                  <h4>silica drug development devices</h4>
                  <h4>individualization of therapy</h4>
                  <h3>application of engineering methods</h3>
                  <h3>systems biology</h3>
                  <h3>genetic changes</h3>
                  <h3>computational methods</h3>
                  <h2>Bioengineering</h2>
                  <p>epithelial morphogenesis</p>
                  <h2>regeneration</h2>
                  <h3>software engineering</h3>
                  <p>network based modeling for cancer pathways</p>
                  <h4>experimental and computational approaches</h4>
              </div>
              <div class="people centered"><h1>Our People</h1></div>
          </div>
        </div>
      </div>
      <div class="container">
        <div class="row huntRow">
          <div class="col-xs-12 col-sm-5 col-md-3">
            <img class="img-responsive huntPic" src="img/huntBioPhoto.jpg" alt="C Anthony Hunt Pic">
          </div>
          <div class="col-xs-12 col-sm-7 col-md-9 huntContainer">
            <h3>Principal Investigator</h3>
            <h4><a href="https://pharmacy.ucsf.edu/c-anthony-hunt" target="_blank">C. Anthony Hunt</a></h4>
            <p class="huntInfo">Dr. Hunt is a theoretical systems pharmacologist who develops advanced simulation methods to improve multiscale mechanistic explanations of complex phenomena in the presence and absence of therapeutic interventions.</p>
            <p class="huntInfo huntBio">Dr. Hunt&#39;s research is motivated by this question&#58; What are the causal mechanisms that link induced changes in molecular level events to emergent changes in phenotype at the organism level? We need new ways to explore and challenge plausible answers. So doing will facilitate moving away from empirical approaches to biomedical research and reliance on correlational methods toward new approaches based on improving mechanistic knowledge and insight.<br><a class="moreHunt" href="#">Learn More</a></p>
            <p class="huntInfo huntBioContinued">
            We need ways to build and challenge experimentally &#40;falsify&#41; many alternative, nested, networked mechanistic hypotheses. Doing so using established, bottom-up, computational mathematical models is problematic at best. New methods and new approaches are needed. My search started in earnest circa 1997. We have now developed and demonstrated the scientific power of a novel, fundamentally new class biomedical simulation models along with methods to challenge and iteratively improve plausible, explanatory mechanisms. We adapted concepts and advanced methods from several domains. I call the process the synthetic method of modeling and simulation (SM). I say synthetic because standalone biomimetic components are plugged together in specific ways to form hierarchical, multiscale biomimetic mechanisms in software. I refer to the models as analogues because even though they exist only in silico, their phenotypes can be made to increasingly overlap more of the referent, biological phenotype. They are designed to evolve to become embodiments of most of what we know (or think we know) about aspects of specific organisms and their components.</p>
          </div>
        </div>
        <div class='row'>
          <div class='col-xs-12'>
            <div class="staffInfo">
              <h3>Staff</h3>
              <h4>Glen E. P. Ropella</h4>
              <p>Glen is a Research Scientist with oversight for software development in the group. He is experienced in multiple aspects of software engineering, modeling and simulation. He is an expert in the design and execution of agent-based models and an expert in the use of the Swarm platform.</p>
              <p class="email"><img class="mailIcon" src="img/mail.png" alt="mailImg"> gepr [at] tempusdictum [dot] com </p>
              </p>
            </div>
          </div>
          <div class='col-xs-12'>
            <div class="staffInfo">
              <h3>Visiting Scientists</h3>
              <h4>Joe LaPointe</h4>
              <p class="email"><img class="mailIcon" src="img/mail.png" alt="mailImg"> lapointeconsulting [at] att [dot] net </p>
              <p>Visiting Professor of Systems Biology</p>
          </div>
          </div>
          <div class='col-xs-12'>
            <div class="staffInfo">
              <h3>Postdoctoral Researchers</h3>
              <h4>Sean H. J. Kim</h4>
              <p>Sean is a postdoctoral researcher, with expertise in multiscale simulation modeling and systems biology. His current research focuses on developing new computational methods and tools for drug development, and modeling epithelial morphogenesis, regeneration, and malignancy.</p>
              <p class="email"><img class="mailIcon" src="img/mail.png" alt="mailImg"> seanhjk [at] gmail [dot] com </p>
            </div>
          </div>
          <div class='col-xs-12'>
            <div class="staffInfo">
              <h3>Graduate Students</h3>
              <h4>Jesse Engelberg</h4>
              <p>Jesse is a graduate student in the UCSF/UCB Joint Graduate Group in Bioengineering. His research focuses on computational modeling of cell biology, especially multicellular tumor spheroids and epithelial cell morphogenesis. His research interests include computational biology, bioinformatics, and modeling and simulation.</p>
              <p class="email"><img class="mailIcon" src="img/mail.png" alt="mailImg"> jesse [at] amherstcollege [dot] net </p>

              <h4>Mark Grant</h4>
              <p>Mark is a graduate student in the UCSF/UCB Joint Graduate Group in Bioengineering, with research emphasis on computational biology and bioinformatics. His most recent project focused on simulation of mammalian tissue morphogenesis. His interests include modeling of biological processes and protein function determination.</p>
              <p class="email"><img class="mailIcon" src="img/mail.png" alt="mailImg"> mgrant [at] socrates [dot] Berkeley [dot] edu</p>
            </div>
          </div>
          <div class='col-xs-12'>
            <div class="staffInfo">
              <h3>Recent Group Alumni</h3>
              <h4>Amy H Lin</h4>
              <p>Amy H Lin was a postdoc in the Hunt lab with a Ph.D. in mathematics. She worked on a multicellular tumor spheroid model, and she also compared the results of agent-based and equation-based models of different systems.</p>

              <h4>Anita Grover</h4>
              <p>Anita Grover is a graduate student in the Graduate Program in Pharmaceutical Sciences & Pharmacogenomics at UCSF. She completed a rotation in the Hunt Lab, with research on in silico drug development devices.</p>

              <h4>Christine Case</h4>
              <p>Christine Case is interested in the individualization of therapy with potentially toxic drugs like chemotherapy agents. Her research included modeling the processes of drug elimination and the impact of patient-specific information and health status on those processes, specifically with an emphasis on inflammation. Christine is a graduate from UC Berkeley in Bioengineering.</p>

              <h4>Cindy Chen</h4>
              <p>Cindy Chen was a postgraduate researcher with research on agent based modeling and simulation for cancer, gene database analysis, and network based modeling for cancer pathways. She also has provided with hardware and software support.</p>

              <h4>Craig Furman</h4>
              <p>Craig is a Software Engineer and has worked on the ISL project. He has brought many years of commercial software development experience to the project. He will be starting a PhD in Bioinformatics/Computational Biology soon and would like to focus on the computational issues in Population Genetics, Genome Sequencing and Evolution.
              </p>

              <h4>Debbie Lin</h4>
              <p>Debbie has been involved in modeling systems that are at the intersection of biopharmaceutical science and policy. Specifically, such systems include drug in-licensing processes, drug development pipelines, and biological systems. Her aims include application of engineering methods and a number of simulation tools to explore these topics including agent based modeling software, process manufacturing and decision analytic tools.
              </p>

              <h4>Donglei Hu</h4>
              <p>Donglei Hu is a genetic research specialist at the UCSF Asthma Laboratory.  He is involved in the study of genetic and epgenetic factors in development and treatment of asthma in different ethnic groups.
              </p>

              <h4>Jason Ju</h4>
              <p>Jason was a UCSF MSTP student &#40;incoming Fall 2002&#41; and completed a summer rotation in the Hunt Lab. On rotation, he explored the use of the relatively new agent-based modeling approach in studying and simulating biological systems on the subcellular, cellular and tissue levels. In the long term, he seeks to develop an in silico agent-based model of a system infected or affected by HIV.</p>

              <h4>Keith Erickson</h4>
              <p>Keith Erickson graduated from the University of Washington with a B.S. in Chemical Engineering in 1999. He is a doctoral student in the UCSF/UCB Joint Graduate Group in Bioengineering under Professor Adam Arkin with a generous fellowship from the Whitaker Foundation. His research is focused on elucidating the biochemical control mechanisms behind neutrophil chemotaxis using a combination of experimental and computational approaches.</p>

              <h4>Michael Oakley</h4>
              <p>Michael Oakley&#39;s research is focused on genetic changes that occur in the developing mouse liver, more specifically, how these genetic changes control survival of the organ's vasculature.

              <h4>Sunwoo Park</h4>
              <p>Sunwoo Park was a postdoctoral scholar at the Hunt Lab. He developed high performance distributed/parallel modeling and simulation framework for large-scale biological systems. His expertise draws from the fields of DEVS M&#38;S, discrete event systems, and distributed/parallel computing. His main research interests include discrete event-oriented M&#38;S, mathematical modeling, multi-agent M&#38;S, system biology, and bioinformatics.
              </p>
            </div>
          </div>
          <div class="col-xs-12">
            <div class="staffInfo">
              <h3>Recent Graduates</h3>
              <h4>Abraham Anderson</h4>
              <p>Abraham Anderson is a doctoral graduate who has been developing network-based analyses of diverse biological systems to support therapeutic target and biomarker discovery. He was affiliated with the Bioengineering Graduate Group, jointly administered by UCSF and UC Berkeley.</p>

              <h4>Amina Qutub</h4>
              <p>Amina Qutub is a doctoral graduate from the UCSF/UCB Joint Graduate Group in Bioengineering, with a background in chemical engineering and an emphasis on mathematical modeling and neurobiology. Her thesis research focused on simulating the physiology of the blood-brain barrier, with the long-term goal of improving drug development and delivery for neurological diseases.</p>

              <h4>Jon Tang</h4>
              <p>Jon is a doctoral graduate from the UCSF/UCB Joint Graduate Group in Bioengineering. He is interested in computer modeling of biological systems, in particular various aspects of the immune system. He is contstructing an agent-based model of the leukocyte adhesion cascade and leukocyte transendothelial migration during inflammation. </p>

              <h4>Lan Xia</h4>
              <p>Lan Xia is a doctoral graduate from the comparative biochemistry program at UC Berkeley; she has worked on the mammary gland development and patterning formation project, together with Mark Grant. Her goal is to build up a multi-model biodevice for morphodynamic process applied in mammary gland morphogenesis.</p>

              <h4>Li Yan</h4>
              <p>Li Yan is a doctoral graduate from the UCSF/UCB Joint Graduate Group in Bioengineering. Her long-term research goal is to develop simulation techniques that can facilitate and guide drug development, as well as facilitate optimization of drug-based therapies. Her thesis research focused on developing Agent-Based Models (ABMS) of Isolated, Perfused Rat Liver (IPRL).</p>

              <h4>Robert Otillar</h4>
              <p>Robert Otillar is a doctoral graduate who conducted data-intensive computational biology research on protein alignments and annotation and dentified informative data-subsets and patterns by using biological and statistical criterion.</p>

              <h4>Shahab Sheikh-Bahaei</h4>
              <p>Shahab is a doctoral graduate from the UCSF/UCB Joint Graduate Group in Bioengineering. His areas of interest include modeling and simulation, multi-agent systems, self-organizing systems (complexity theory) and artificial intelligence.</p>

              <h4>Tedd Lam</h4>
              <p>Teddy is a doctoral graduate from the Graduate Program in Pharmaceutical Sciences &#39; Pharmacogenomics at UCSF. His research includes development of computational tools and methods to better understand and predict drug-drug interaction.</p>

              <h4>Yu Liu</h4>
              <p>Yu Liu is a doctoral grauduate from the UCSF/UCB Joint Graduate Group in Bioengineering. Her research focued on drug absorption across small intestine epithelia barrier, a complicated process. To investigate the feasibility of an in silico experiment device to study this process, she used a new computational strategy, agent based modeling, with a goal to develop in silico devices to help expedite the drug development process.</p>

              <h4>Yuanyuan Xiao</h4>
              <p>Yuanyuan is a doctoral graduate from the biopharmaceutical sciences at UCSF. Her research interests are microarray data analysis, and generally statistical methods in bioinformatics. Yuanyuan says she still has a lot to learnâ€¦</p>
            </div>
          </div>
        </div>
      </div>



          <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
          <script src="js/jquery-migrate-1.4.1.min.js"></script>
          <!-- Latest compiled and minified JavaScript -->
          <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
          <script type="text/javascript" src="slick/slick.min.js"></script>
          <script src="js/people.js"></script>

      </body>
    </html>
